site/content/contribute/more-info/mobile/push-notifications/service.md

```diff
@@ -89,9 +89,9 @@ For the sake of making this guide simple we located the files at `/home/ubuntu/m
     ```
 - Extract the private key from the certificate you exported ..
     ```sh
-    $ openssl pkcs12 -in Certificates.p12 -out aps_production_priv.pem -nodes -clcerts -passin pass:
+    $ openssl pkcs12 -in Certificates.p12 -out aps_production_priv.pem -nodes -clcerts -passin pass: -legacy
```

Comment:
Since using the `-legacy` flag changes the default algorithms, maybe is better if we also specify the algorithm we want in the command? To avoid problems for people using different OpenSSL versions thus potentially providing different defaults. What do you think?

Reply:
Good idea.